### PR TITLE
1.x-branch pylint cleanup

### PR DESCRIPTION
**Pull request by bcl**
Fixed everything except a couple LVMSnapShotBase errors and the intermittent element error.

### CODE REVIEW

File: blivet/size.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def __mul__(self, other, context=None):
         return Size(Decimal.__mul__(self, other))
     __rmul__ = __mul__
 
-    def __div__(self, other, context=None):
+    def __div__(self, other, context=None):             # pylint: disable=unused-argument
```
Comment (vpodzime) [inline blivet/size.py]: why so many spaces?

**Comment by vpodzime** [comment]
Looks good to me.

File: blivet/size.py (hunk 1 above)
Comment (bcl) [inline blivet/size.py]: > why so many spaces? "Two ought to be enough." :)

makes it easier to read.

**Comment by dwlehman** [comment]
I think we could use 25afe59 ("Cleanup wildcard...") and fb45fe1 ("Include python3-bugzilla...") on master, too.

File: blivet/size.py (hunk 1 above)
Comment (vpodzime) [inline blivet/size.py]: Fair enough :)

**Comment by bcl** [comment]
Pushed to 1.x-branch and cherry-picked those 2 to master.

**Pull request closed without merging**